log-collector-script/linux/eks-log-collector.sh

```diff
@@ -800,8 +801,14 @@ get_io_throttled_processes() {
   ok
 }
 
+get_reboot_history() {
+  try "collect reboot history"
+  timeout 75 last reboot > "${COLLECT_DIR}"/system/last_reboot.txt 2>&1 || echo -e "\tTimed out, ignoring \"reboot history output \" "
```

Comment:
I usually use this:
```
journalctl --list-boots
```

any advantage to `last reboot`?

Reply:
Yeah, `last reboot` will include the status whereas `journalctl --list-boots` won't. In our case it's relevant because we can see *2 `still running` statuses, which indicates that there was ungraceful shutdown and not just a regular reboot.

Reply:
Makes sense 👍